### PR TITLE
Fix small errors with clasp clone and clasp logs

### PR DESCRIPTION
**Pull request by campionfellin**
Thanks to @ajlai for finding the array of `{name:_,value:_}` in Inquirer.

Signed-off-by: campionfellin <campionfellin@gmail.com>

Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [x] `npm run test` succeeds.
- [x] `npm run lint` succeeds.
- [ ] Appropriate changes to README are included in PR.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -300,19 +300,17 @@ commander
             q: 'mimeType="application/vnd.google-apps.script"',
           });
           const files = data.files;
-          const fileIds:string[] = [];
+          const fileIds: Array<{ name: string; value: string; }> = [];
           if (files.length) {
             files.map((file: any) => {
-              fileIds.push(`${file.name}`.padEnd(20) + ` - (${file.id})`);
+              fileIds.push({name: `${file.name}`.padEnd(20) + ` - (${file.id})`, value: `${file.id}`});
```
Comment (grant) [inline index.ts]: I think this should be formatted on multiple lines.

File: index.ts [hunk 2]
```diff
@@ -300,19 +300,17 @@ commander
             q: 'mimeType="application/vnd.google-apps.script"',
           });
           const files = data.files;
-          const fileIds:string[] = [];
+          const fileIds: Array<{ name: string; value: string; }> = [];
```
Comment (grant) [inline index.ts]: Can you not do this?
```ts
const fileIds: {
  name: string;
  value: string;
}[] = [];
```

File: index.ts (hunk 1 above)
Comment (grant) [reply index.ts]:
```ts
fileIds.push({
  name: `${file.name.padEnd(20)} - (${file.id})`,
  value: file.id,
});
```

File: index.ts (hunk 2 above)
Comment (campionfellin) [reply index.ts]: `tslint` gives this error: `Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.`

Comment (campionfellin) [reply index.ts]: https://github.com/google/clasp/blob/60393ef2f693f6f2cfa7b2b63cbbed1fa0f8438e/tslint.json#L5

File: index.ts (hunk 1 above)
Comment (ajlai) [reply index.ts]: why is this not just
```ts
            const fileIds = files.map((file) => {
              return {
                name: `${file.name.padEnd(20)} - (${file.id})`,
                value: file.id
              };
            });
``` 
?
Typescript can figure out the type here, no need to explicitly state it.

Comment (grant) [reply index.ts]: 👍 This would be an even better refactor.

Comment (campionfellin) [reply index.ts]: Perfect, this has now been fixed.

**Comment by campionfellin** [comment]
I also noticed that `clasp clone` does not check if a `.clasp.json` file already exists in the directory. Do we want to do that?

**Comment by grant** [comment]
Yes.
- `clasp clone` should fail if there is an existing `.clasp.json` file
- There should be a test case for this.

**Pull request merged**